.github/workflows/lint.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint Checks
+on: [push, pull_request]
```

Comment:
We can probably do only on: [pull_request] to avoid two runs. pull_request is enough for the tests to run in every push once the PR is created

Reply:
looks like there's only 2 runs: https://github.com/oppia/oppia/actions so we should be fine.

Reply:
As discussed, we will keep push now and change it in a follow up to push on branch develop to avoid two runs for any branches created off the main repo. This will be in a follow up